README.md

```diff
@@ -308,8 +308,70 @@ just added.
 ## API
 
 <generated_api_start />
+#### The Jest global environment
+
+In your test files, Jest puts each of these methods and objects into the global environment. You don't have to require or import anything to use them.
+
+  - `afterEach(fn)`
+  - `beforeEach(fn)`
+  - [`check`](https://facebook.github.io/jest/docs/api.html#check)
+  - `describe(name, fn)`
+  - [`expect(value)`](https://facebook.github.io/jest/docs/api.html#expect-value)
+  - [`gen`](https://facebook.github.io/jest/docs/api.html#gen)
+  - `it(name, fn)`
+  - `fit(name, fn)` executes only this test. Useful when investigating a failure
+  - [`jest`](https://facebook.github.io/jest/docs/api.html#the-jest-object)
+  - `require(module)`
+  - [`require.requireActual(moduleName)`](https://facebook.github.io/jest/docs/api.html#require-requireactual-modulename)
+  - [`require.requireMock(moduleName)`](https://facebook.github.io/jest/docs/api.html#require-requiremock-modulename)
+  - `test(name, fn)` is an alias for `it`
+  - `xdescribe(name, fn)`
+  - `xit(name, fn)`
+
+#### Writing assertions with `expect`
+
+When you're writing tests, you need to check that values are what you
+expect all the time. That's what you use `expect` for.
+
+  - [`expect(value)`](https://facebook.github.io/jest/docs/api.html#expect-value)
+  - [`.lastCalledWith(arg1, arg2, ...)`](https://facebook.github.io/jest/docs/api.html#lastcalledwith-arg1-arg2)
```

Comment:
It's annoying to have to look around for `toBe` and `toEqual`, but totally missing a method because you don't find it is worse. I think in an "API reference" it's better to be alphabetical, but in a "tutorial" or "guide" it's better to go most-common-first. Because when you go to the API reference, it's not because you want an intro to this whole thing, it's more likely because you have a specific question.

E.g. look at https://developer.android.com/reference/android/net/package-summary.html